docs/messages.json

```diff
@@ -70,6 +70,61 @@
           "http://messages.beamos.mr-beam.org/img/Testimonial%20EN.png"
         ]
       }
+    },
+    {
+      "id": 3,
+      "date": "27. 07. 2022",
+      "restrictions": {
+        "__": "Evaluation  of restrictions needs to be true to show the message",
+        "channels": [
+          ""
```

Comment:
shouldn'T this be showing only for stable users? 
Even though the version restriction will take care of this, I'd prefer to restrict this from Beta here